server/server/models/categories.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy import Column, String, Integer
+from sqlalchemy.orm import relationship
+from . import Base
+
+
+class Category(Base):
+    """SQLAlchemy model for table categories"""
```

Comment:
According to [PEP-257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings):

> Insert a blank line after all docstrings (one-line or multi-line) that document a class -- generally speaking, the class's methods are separated from each other by a single blank line, and the docstring needs to be offset from the first method by a blank line.